packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -582,6 +588,17 @@ - (instancetype)initWithRegistry:(NSObject<FlutterTextureRegistry> *)registry
 }
 
 - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result {
+  if (!_dispatchQueue) {
+    _dispatchQueue = dispatch_queue_create("FlutterCameraQueue", NULL);
```

Comment:
Why not initialize this in the `initWithRegistry:messenger:` method?
I guess the cost of initializing the queue is low even if it is never used is low, so I don't think we need to initialize it lazily?

Reply:
How do I get this merged into the flutter mainline?

Reply:
I think @bparrishMines is the one to land this if he agrees.

Reply:
@bparrishMines ?